frontend/src/containers/NewGroup.js

```diff
@@ -244,6 +249,7 @@ export class NewGroup extends Component {
       mission: attr.mission,
       longDescription: attr.contribute ? `${attr.description}\n\n# Contribute\n\n${attr.contribute}` : attr.description,
       website: attr.website,
+      HostId: hostCollective.settings.HostId,
```

Comment:
I've changed the default value of `isActive` to false in the `Group` model in the API.